### PR TITLE
chore(localstack): add extraLabels option and align documentation

### PR DESCRIPTION
**Pull request by levivannoort**
## Motivation

We're using `Localstack` when spinning up temporary pull request environments. I would like to use `Kyverno` to act upon a time-to-live (ttl) label set on all of the resources with the temporary pull request environments. Although it had a an `extraAnnotations` option, I am missing an `extraLabels` option.

Another option would be to use the `common.labels` of the chart dependency.

## Changes

Added a `extraLabels` value which is evaluated through the template.

## Testing

Add a key & value under the `extraLabels`, an example can be see below with the `Kyverno` time-to-live (ttl) label.

```
extraLabels:
    cleanup.kyverno.io/ttl: 2m
```

### CODE REVIEW

File: charts/localstack/README.md [hunk 1]
```diff
@@ -92,9 +92,12 @@ The following table lists the configurable parameters of the Localstack chart an
...
 | `podSecurityContext`                                 | Enable security context for Localstack pods                                                                                                                                                                                           | `{}`                                                    |
+| `extraDeploy`                                        | Extra objects to deploy (value evaluated as a template)                                                                                                                                                                               | `{}`                                                    |
+| `extraAnnotations`                                   | Add additional annotations to every resource (value evaluated as a template)                                                                                                                                                          | `{}`                                                    |
```
[the remark this answers is not in the document]
Comment (alexrashed) [reply charts/localstack/README.md:99]: Sure, that's on the bucket list 😄

File: charts/localstack/templates/_helpers.tpl [hunk 2]
```diff
@@ -40,6 +40,9 @@ helm.sh/chart: {{ include "localstack.chart" . }}
 app.kubernetes.io/version: {{ .Chart.AppVersion | quote }}
 {{- end }}
 app.kubernetes.io/managed-by: {{ .Release.Service }}
+{{- if .Values.extraLabels }}
+{{ toYaml .Values.extraLabels }}
```
[the remark this answers is not in the document]
Comment (levivannoort) [reply charts/localstack/templates/_helpers.tpl:44]: I'd probably opt for going with the `extraLabels` to indeed have consistent usage in the chart for now, and later adopt the `common` chart with more of the template options if needed.

File: charts/localstack/templates/_helpers.tpl [hunk 3]
```diff
@@ -40,6 +40,9 @@ helm.sh/chart: {{ include "localstack.chart" . }}
...
+{{- end }}
```
[the remark this answers is not in the document]
Comment (levivannoort) [reply charts/localstack/templates/_helpers.tpl]: Aligned the behavior for `extraLabels` with `extraAnnotations` to have consistency, thanks for pointing it out.

File: charts/localstack/README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (levivannoort) [reply charts/localstack/README.md:99]: If I have some additional time I could create a pull request for it!

[the remark this answers is not in the document]
Comment (alexrashed) [reply charts/localstack/README.md:99]: That would be great :)

**Review by alexrashed — APPROVED** [review]
Thanks a lot for addressing the comments! 🚀
And congrats to PR # 2^7! 🥳 
I'll merge this directly and create a new release :)

**Pull request merged**